src/legacy/core_plugins/elasticsearch/lib/ensure_es_version.js

```diff
@@ -48,7 +48,11 @@ export function ensureEsVersion(server, kibanaVersion) {
     ]
   })
     .then(function (info) {
-    // Aggregate incompatible ES nodes.
+      if (process.env.UNSAFE_RUN_AS_VERSION) {
+        kibanaVersion = process.env.UNSAFE_RUN_AS_VERSION;
```

Comment:
Instead of faking the version, I would rather we downgrade the error.

We could use an environment variable like `UNSAFE_IGNORE_ES_VERSION` to signal that we should still startup, however, we should log a notice that we are skipping the ES check due to this environment variable.

Reply:
Probably a better solution would be to add a development only configuration option to the elasticsearch plugin schema which allows for ignoring the ES version check.